backend/workflow_manager/endpoint_v2/source.py

```diff
@@ -488,8 +501,50 @@ def add_input_from_api_storage_to_volume(self, input_file_path: str) -> None:
         """Add input file to execution directory from api storage."""
         infile_path = os.path.join(self.execution_dir, WorkflowFileType.INFILE)
         source_path = os.path.join(self.execution_dir, WorkflowFileType.SOURCE)
-        shutil.copyfile(input_file_path, infile_path)
-        shutil.copyfile(input_file_path, source_path)
+        if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
```

Comment:
@muhammad-ali-e Is this code snippet manually implementing a copy() ? Will it be easier for sdk to support copy as one of the APIs of FileStorage?

Reply:
One more question. But I m not sure if this is the right context. Where do we create this EXTRACT file? Looks like we might have to use local storage as provider for that file as it is of type Temporary in Single Container @muhammad-ali-e

Reply:
I've created this copy method in the SDK within a utility class. You can find the details here https://github.com/Zipstack/unstract-sdk/pull/119